### PR TITLE
Completed lesson 067

### PR DESCRIPTION
**Pull request by Greanncy**
@nehero haha thanks for fixing my problem I'm doing my best to catch up

### CODE REVIEW

File: lessons/06-html-layout/posts/post-1.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+            voluptatum, excepturi adipisci?</p>
+        <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Doloribus eos excepturi illum assumenda quas ullam
+            inventore. Esse perspiciatis quibusdam alias architecto culpa, optio officiis dolore repellat repudiandae
+            voluptatum, excepturi adipisci?</p>
+            <hr />
+    </article>
+    <footer>
+        <p>- Greanncy Ramales</p>
+    </footer>
+</main>
```
Comment (ozziexsh) [inline lessons/06-html-layout/posts/post-1.html:40]: Oops! This looks good except we need to check where our `<body>` tag is at. Our body tag needs to wrap **all** content that is displayed on the website, so we need to place the `<header>` and `<main>` tags here inside of it

**Review by ozziexsh — COMMENTED** [review]
One small thing but other than that it's great!

**Comment by Greanncy** [comment]
Hi, so the <main> and <header> tag is suppose to be inside the <body> tag?

**Comment by ozziexsh** [comment]
@Greanncy Yup! Everything that you want the user to see should be inside of the `<body>` tag

Every html page should have the format of

```html
<html lang="en">
<head>
  <title>Document</title>
</head>
<body>
  ... all of your visible content here
</body>
</html>
```

**Pull request merged**